src/components/ProjectCard/index.jsx

```diff
@@ -39,6 +39,12 @@ export default class ProjectCard extends Component {
     description: null,
   };
 
+  handleDescriptionClick = event => {
```

Comment:
It looks like this should now be `summary` instead of `description` according to https://github.com/mozilla-frontend-infra/codetribute/pull/52. Let's reflect that here as well.

`handleSummaryClick`.